docs/Applications/Docs/Style Guide.md

```
@@ -0,0 +1,37 @@
+#### Ordered Lists
+
+- Ordered lists must be in `1. 1. 1.` format.
+
+#### Unordered Lists
+
+- Unordered list items must use dashes (`-`) in front of line items.
+
+### Links and Aliases
+
+- When linking using `[[wikilink]]` syntax, if used in the middle of a sentence, alias the link to fit the flow of the sentence. For example, `[[Team]]` is often aliased as `team` to maintain sentence structure. Links can be aliased like so: `[[Team|team]]`.
+	- This applies to capitalization and wording of the link entirely. Consider the following example: `"Read our [[Setup For Local Use|setup guide]] for more information."`
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)